packages/web/src/pages/asset-detail-page/components/mobile/ArtistCoinDetailsPage.tsx

```diff
@@ -0,0 +1,184 @@
+    'The total number of your artist coins that will ever exist. This amount is fixed and never changes.',
+  marketCap:
+    'The current total value of all your artist coins in circulation, calculated by multiplying the current price by the total supply.',
+  fdv: 'The theoretical market cap if all tokens were in circulation, calculated by multiplying the current price by the total supply.',
+  price: 'The current price of a single artist coin in USD.',
+  liquidity:
+    'The amount of funds available for trading your artist coin, which affects how easily it can be bought or sold.',
+  circulatingSupply:
+    'The number of artist coins currently available for trading, excluding any tokens that are locked or reserved.'
+}
+
+export const ArtistCoinDetailsPage = () => {
+  const location = useLocation()
+  // Locations should be in the format /coins/:ticker/details (COIN_DETAIL_ROUTE)
+  const ticker = location.pathname.split('/')[2]
+  const { data: artistCoin } = useArtistCoinByTicker({ ticker: ticker ?? '' })
+  const { spacing } = useTheme()
+
+  return (
+    <MobilePageContainer fullHeight>
+      <Flex column w='100%' backgroundColor='white'>
+        <Flex direction='column' gap='l' p='l'>
+          {/* Token Info Section */}
+          <Flex direction='column' gap='m'>
+            <Flex alignItems='center' gap='m'>
+              {/* Token Icon */}
+              <TokenIcon
+                logoURI={artistCoin?.logoUri}
+                w={spacing['4xl']}
+                h={spacing['4xl']}
+                hex
+              />
+              <Flex direction='column' gap='xs'>
+                <Text variant='title' size='l'>
+                  {artistCoin?.name ?? messages.unknownToken}
+                </Text>
+                <Text
+                  variant={artistCoin?.name ? 'body' : 'title'}
+                  size={artistCoin?.name ? 'm' : 'l'}
+                  color={artistCoin?.name ? 'subdued' : 'default'}
+                >
+                  {artistCoin?.ticker ?? messages.unknownTicker}
+                </Text>
+              </Flex>
+            </Flex>
+          </Flex>
+
+          <Divider />
+
+          {/* Coin Address */}
+          <Flex direction='column' gap='xs'>
+            <Text variant='label' size='s' color='subdued'>
+              {messages.coinAddress}
+            </Text>
+            <Text variant='body' size='s' color='default'>
+              {artistCoin?.mint ?? messages.unknown}
+            </Text>
+          </Flex>
+
+          {/* On-Chain Description */}
+          <Flex direction='column' gap='xs'>
+            <Text variant='label' size='s' color='subdued'>
+              {messages.onChainDescription}
+            </Text>
+            <Text variant='body' size='s' color='default'>
+              {artistCoin?.description ??
+                `${artistCoin?.ticker ?? 'UNKNOWN'} is an artist coin created on Audius. Learn more at https://audius.co/coin/${artistCoin?.ticker ?? 'unknown'}`}
```

Comment:
this is kinda weird to fallback to UNKNOWN? maybe should just null check artistCoin.ticker above

Reply:
this is all the same code as the ArtistCoinDetailsModal so keeping everything the same. could probably move it to a content component to share between the two

Reply:
yeah @Kyle-Shanks i updated detailsmodal a bunch in another pr, this is fine for now and we can follow up on this ui right after, dont worry for now though